examples/step-27/doc/intro.dox

```diff
@@ -312,27 +312,30 @@ Assuming that the cell $K$ is not degenerate, i.e. that the mapping from the
 unit cell to cell $K$ is sufficiently regular, above condition is of course
 equivalent to
 @f[
-	\int_{\hat K} |\nabla^s \hat u(\hat{\bf x})|^2 \; d\hat{\bf x} < \infty
+	\int_{\hat K} |\nabla^s \hat u(\hat{\bf x})|^2 \; d\hat{\bf x} < \infty\,,
 @f]
-where $\hat u(\hat{\bf x})$ is the function $u({\bf x})$ mapped back onto the unit cell
+where $\hat u(\hat{\bf x})$ is the function $u({\bf x})$ mapped back onto the reference unit cell
```

Comment:
Is there a reason you say 'reference unit cell' now? Isn't it either the reference cell or the unit cell?